src/components/Button/PlayButton.js

```diff
@@ -15,15 +26,32 @@ const StyledPlayButton = styled.div`
     position: relative;
     left: 3px;
   }
-  ${props => props.hover ? 'opacity: 0;' : ''},
+  ${props => props.hover ? 'opacity: 0;' : ''}
+  ${props => 
+    props.type === 'lesson' ||
+    (props.type === 'course' && (props.expanded === false || props.expanded === 'horizontal'))
+     ? `${centerButtonRules}`
+     : ''
+   }
 `
 
-const PlayButton = ({ hover=false, className }) => {
-  return <StyledPlayButton hover={hover} className={`${hover ? `${hoverPlayBtnClasses} card-play-btn` : playBtnClasses} ${className}`}/>
+const PlayButton = ({ hover=false, type, expanded, className }) => {
+  return <StyledPlayButton hover={hover}
+            className={`${hover
+                          ? `${hoverPlayBtnClasses} ${hoverColorClasses[type] ? hoverColorClasses[type] : ''} card-play-btn`
+                          : playBtnClasses
+                         }
+                        ${type === 'playlist' ? `${hoverColorClasses[type]}` : ''}
```

Comment:
`<PlaylistCard>`'s `PlayButton` is always visible, but it does get a color on hover